EngineeringModel/EngineeringModelModule.cs

```diff
@@ -129,6 +133,11 @@ public EngineeringModelModule(IRegionManager regionManager, IFluentRibbonManager
         /// </summary>
         internal IParameterActualFiniteStateListApplicationBatchService ParameterActualFiniteStateListApplicationBatchService { get; private set; }
 
+        /// <summary>
+        /// The <see cref="IChangeOwnerShipBatchService"/> used to change the ownership of multiple <see cref="IOwnedThing"/>s in a batch operation
+        /// </summary>
+        internal IChangeOwnerShipBatchService ChangeOwnerShipBatchService { get; private set; }
```

Comment:
The "S" should not be capital in "Ownership" is the case of class, interface and variable names.